podcasts/End of Year/EndOfYear.swift

```diff
@@ -206,9 +206,13 @@ struct EndOfYear {
     }
 }
 
+extension EndOfYear {
+    static var defaultDuration = 10.seconds
+}
+
 class StoriesHostingController<ContentView: View>: UIHostingController<ContentView> {
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        .lightContent
+        .darkContent
```

Comment:
Same here, can we switch on the current year like above?

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2393/commits/51402ee5e284bb2292bb1c5cad8617c06991143e